internal/util/util_test.go

```diff
@@ -157,3 +159,53 @@ func (t *UtilTest) TestResolveConfigFilePaths() {
 	AssertEq(nil, err)
 	ExpectEq(filepath.Join(homeDir, "test.txt"), mountConfig.LogConfig.FilePath)
 }
+
+func (t *UtilTest) TestStringifyShouldReturnAllFieldsPassedInCustomObjectAsMarshalledString() {
+	sampleMap := map[string]int{
+		"1": 1,
+		"2": 2,
+		"3": 3,
+	}
+
+	sampleNestedValue := nestedCustomType{
+		SomeField: 10,
+		SomeOther: sampleMap,
+	}
+	customObject := &customTypeForSuccess{
+		Value:       "test_value",
+		NestedValue: sampleNestedValue,
+	}
+
+	actual := Stringify(customObject)
+
+	expected := "{\"Value\":\"test_value\",\"NestedValue\":{\"SomeField\":10,\"SomeOther\":{\"1\":1,\"2\":2,\"3\":3}}}"
+	AssertEq(expected, actual)
+}
+
+func (t *UtilTest) TestStringifyShouldReturnEmptyStringWhenMarshalErrorsOut() {
+	customInstance := customTypeForError{
+		value: "example",
+	}
+
+	actual := Stringify(customInstance)
+
+	expected := ""
+	AssertEq(strings.TrimSpace(expected), strings.TrimSpace(actual))
```

Comment:
Why TrimSpace is required here?

Reply:
its typo... not required, have removed it